### ISSUE TITLE
Load lyrics from text files

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#1008]
_From [Cyril.Santune](https://code.google.com/u/101650448614500705618/) on November 29, 2010 14:14:39_

I have simple txt files with .lrc extension. It should be possible read it directly in the Clementine player.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=1008_

**Comment by DoctorSubtilis** [comment]
I agree: Clementine must read lyric from local file (and/or from ID3 tag)

**Comment by brutalcrozt** [comment]
@ richardson183
> well, I think I have better idea, save it directly on the mp3 files.

@ komzar
"Some people don't have strong internet connection. :)"
Well, I am counted on that statement, I am too poor to be alive.

@ alphadeltapapa
Kindly elaborate "embedded metadata tags" :)

I am taking a side on preventing "file pollution", I already save lyrics file years ago,
and it's so blindingly ugly ffs. Anyhow, I am pointing this to be an option, when future disagreements happen.

**Comment by uptonking** [comment]
I really want this feature for offline lyrics

**Comment by DoctorSubtilis** [comment]
But Clenetine can already read a locally embedded lyrics.

**Comment by uptonking** [comment]
embedded lyrics in id3 tags may not always  be able to be  read and shown in the left panel. 
[Here](https://github.com/uptonking/assets/tree/main/audio) are my favorite music in `.m4a`, but embedded lyrics cannot be read.

**Comment by DoctorSubtilis** [comment]
I didn't know m4a, sorry. With all my mp3 embedded lyrics are shown corrrectly...

**Comment by uptonking** [comment]
my bad. After I upgrade clementine to the latest rc version in the release page, lyrics in `.m4a` file tags can be read now.

**Comment by brutalcrozt** [comment]
well still a valid issue tho, interestingly people add lyrics on youtube desc, and youtube-dl can grab it.

**Comment by sukrosono** [comment]
I think I need someone who has knowledge about the mp3 meta tag and the mp4a .
When it defined I can start working in a PR.
